frontend/libs/studio-components/package.json

```diff
@@ -8,8 +8,6 @@
     "lint": "eslint \"src/components/**/*.ts*\""
   },
   "dependencies": {
-    "@digdir/design-system-react": "0.38.0",
-    "@digdir/design-system-tokens": "0.11.2",
```

Comment:
Why are we deleting these dependencies here? Are they not in sync with the version(s) present in other packages/roots? Or has Renovate failed to update them consistently across the monorepo?

I am reflecting on the significant purpose of packages, which is to encompass all necessary dependencies for the package to run independently.

There should be no need to add them again later if one wishes to move this to a separate package in a remote location.

Yarn is intelligent enough to avoid installing the same dependency twice in a monorepo, provided that the packages are on the same version, as Renovate is expected to ensure.

Reply:
I've read the description on the pull request, and I believe the good approach is to ensure that the packages are synchronized within the monorepo. Then we are building up on the principle of standalone packages. Then it's also easy to open the `package.json` file to see which packages this package depends on. 

I prefer to limit the number of packages at the root level and instead ensure that each package has its own dependencies to clarify which dependencies the packages rely on.

Reply:
I find that it is much more convenient to control global dependencies like these in one place, otherwise they will just become out of sync with each other when we update them manually. And that is what has happened here - the rest of the project had the latest versions (0.46 of the React package and 0.12 of the tokens), while this was stuck on 0.38 and 0.11. Although it didn't cause problems, we thought the whole project used the latest version. The root package.json was a lie. There's no reason to specify the same dependency in multiple places when we know that we will always use the same version everywhere.

Also, we can't expect everyone to know about every package.json file downwards in the file structure. If we at some point in time were to move the package out, we could simply follow the file tree upwards to find all the necessary dependencies.